src/main/kotlin/Decorator/littlekai/base/IngredientDecorator.kt

```diff
@@ -1,7 +1,7 @@
 package Decorator.littlekai.base
 
 abstract class IngredientDecorator (private val noodles: Noodles) : Noodles by noodles {
-  protected abstract val COST : Double
+  abstract val COST : Double
```

Comment:
You have found the first noob problem when you start to write test. You are discovered that your solution isn't right because for test something you are refactor production code. 👎 It's normal.